gcloud/bigquery/job.py

```diff
@@ -927,6 +928,11 @@ def __init__(self, name, query, client):
     https://cloud.google.com/bigquery/docs/reference/v2/jobs#configuration.query.useQueryCache
     """
 
+    user_defined_function_resources = _TypedProperty('user_defined_function_resources', list)
```

Comment:
pep8 doesn't like the long line.  Please wrap to < 79 characters (e.g., just after `_TypedProperty`).